### PR TITLE
refactor: 💡 Move SQFormReadonly stories and tests to typescript

### PR DESCRIPTION
**Pull request by hvilander**
Moves stories and tests for read only field to typescript.

### CODE REVIEW

File: stories/SQFormReadOnlyField.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import {Story, Meta} from '@storybook/react';
 
 import {SQFormReadOnlyField as SQFormReadOnlyFieldComponent} from '../src';
+import type {SQFormReadOnlyFieldProps} from 'components/SQForm/SQFormReadOnlyField';
 import {SQFormStoryWrapper} from './components/SQFormStoryWrapper';
 import {createDocsPage} from './utils/createDocsPage';
 import getSizeProp from './utils/getSizeProp';
-import markdown from '../notes/SQFormReadOnlyField.md';
+import type {gridOptions} from './utils/getSizeProp';
```
Comment (hvilander) [inline stories/SQFormReadOnlyField.stories.tsx:9]: Is there any particular reason why this type is not capitalized?
I am happy to update it here.

Comment (SeanGroff) [reply stories/SQFormReadOnlyField.stories.tsx:9]: > Is there any particular reason why this type is not capitalized? I am happy to update it here.

Yea that's be nice. It's also a little odd importing a `type` called `gridOptions` from a utils folder named `getSizeProp`. I don't think this type would be very discoverable.

Might be worth moving to a `types` folder

Comment (20BBrown14) [reply stories/SQFormReadOnlyField.stories.tsx:9]: I believe we made in the _very early_ TS adoption process. We were just babies back then 🍼

File: stories/SQFormReadOnlyField.stories.tsx [hunk 2]
```diff
@@ -46,3 +56,5 @@ WithInitialValue.args = {
 WithInitialValue.parameters = {
   controls: {exclude: 'initialValue'},
 };
+
+export default meta;
```
Comment (SeanGroff) [inline stories/SQFormReadOnlyField.stories.tsx:60]: There should be a newline at the end of the file

**Review by 20BBrown14 — APPROVED** [review]

File: stories/SQFormReadOnlyField.stories.tsx (hunk 2 above)
Comment (hvilander) [reply stories/SQFormReadOnlyField.stories.tsx:60]: :shakes fist at vscode:

File: src/components/SQForm/SQFormRadioButtonGroup.tsx [hunk 3]
```diff
@@ -56,8 +56,6 @@ function SQFormRadioButtonGroup({
     onChange,
   });
 
-  console.log('field', field);
-
```
Comment (hvilander) [inline src/components/SQForm/SQFormRadioButtonGroup.tsx:60]: Found this stray log. Removed it.

**Pull request merged**